vllm/v1/metrics/stats.py

```diff
@@ -52,6 +52,12 @@ class RequestStateStats:
     last_token_ts: float = 0.0
 
 
+@dataclass
```

Comment:
I recognize this is a complete nit, but I think this should go next to `SchedulerStats`. When I originally read this code, since it is between `RequestStateStats` and `FinishedRequestStats`, I thought this would be LoRA stats about a single request, when in reality it is about the global state

Reply:
Yep, makes sense. Fixed